docs/examples/timeline/mode.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <el-radio-group v-model="mode">
+    <el-radio-button label="left" value="left" />
+    <el-radio-button label="alternate" value="alternate" />
+    <el-radio-button label="right" value="right" />
+  </el-radio-group>
+
+  <el-timeline class="mt-4" style="max-width: 600px" :mode="mode">
```

Comment:
```suggestion
  <el-timeline class="mt-4" :mode="mode">
```

This style can be removed to provide a clearer reference.